api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -1851,30 +1841,28 @@ public static String getMessage(String code, Object... args) {
 	 * @param password string that will be validated
 	 * @param systemId system id of the user with password to be validated
 	 * @throws PasswordException
-	 * @since 1.5
-	 * <strong>Should</strong> fail with short password by default
-	 * <strong>Should</strong> fail with short password if not allowed
-	 * <strong>Should</strong> pass with short password if allowed
-	 * <strong>Should</strong> fail with digit only password by default
-	 * <strong>Should</strong> fail with digit only password if not allowed
-	 * <strong>Should</strong> pass with digit only password if allowed
-	 * <strong>Should</strong> fail with char only password by default
-	 * <strong>Should</strong> fail with char only password if not allowed
-	 * <strong>Should</strong> pass with char only password if allowed
-	 * <strong>Should</strong> fail without both upper and lower case password by default
-	 * <strong>Should</strong> fail without both upper and lower case password if not allowed
-	 * <strong>Should</strong> pass without both upper and lower case password if allowed
-	 * <strong>Should</strong> fail with password equals to user name by default
-	 * <strong>Should</strong> fail with password equals to user name if not allowed
-	 * <strong>Should</strong> pass with password equals to user name if allowed
-	 * <strong>Should</strong> fail with password equals to system id by default
-	 * <strong>Should</strong> fail with password equals to system id if not allowed
-	 * <strong>Should</strong> pass with password equals to system id if allowed
-	 * <strong>Should</strong> fail with password not matching configured regex
-	 * <strong>Should</strong> pass with password matching configured regex
-	 * <strong>Should</strong> allow password to contain non alphanumeric characters
-	 * <strong>Should</strong> allow password to contain white spaces
```

Comment:
Like in another comment, please remove any formatting changes that is not relating to the changes in File/Path construction